test_api_models.py

```
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3
+"""
+Test script for Phase 1 models via API endpoints.
+
+Since the models are already loaded by the running server, 
+we'll test them through the API to verify they work correctly.
+"""
+
+import asyncio
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)